### PR TITLE
init code: load directly ui

### LINKED ISSUE #205: Consider loading directly UI files
See https://doc.qt.io/qtforpython/tutorials/basictutorial/uifiles.html, it mentiones QtUiTools class to load directly (dynamically) UI files.

In such way `makeui.sh` script (which calls `pyuic5`) could be dropped.

### PR DESCRIPTION
**Pull request by ljelen**
closes #205

### CODE REVIEW

**Comment by grzanka** [comment]
Fix following errors:
```
./pytripgui/main.py:5:1: F401 'PyQt5.QtWidgets.QMainWindow' imported but unused
./pytripgui/controller/plan_cont.py:53:21: F821 undefined name 'Ui_PlanDialog'
```
see https://travis-ci.org/pytrip/pytripgui/jobs/575147357#L764

**Comment by grzanka** [comment]
Remove calls to makeui.sh script 
https://github.com/pytrip/pytripgui/blob/master/.travis.yml#L58

and delete the script itself.

**Comment by grzanka** [comment]
See this also https://github.com/pytrip/pytripgui/search?q=makeui&unscoped_q=makeui

**Comment by ljelen** [comment]
> Remove calls to makeui.sh script
> https://github.com/pytrip/pytripgui/blob/master/.travis.yml#L58
> 
> and delete the script itself.

**Comment by grzanka** [comment]
Can you check if this method 
https://github.com/pytrip/pytripgui/blob/f05345bb274de8ca7565f32f1b79e78920936fc3/pytripgui/controller/main_cont.py#L453

See also https://github.com/pytrip/pytripgui/blob/f05345bb274de8ca7565f32f1b79e78920936fc3/pytripgui/util.py#L9

can be used instead of relative paths pointing to `ui` files ?

**Comment by grzanka** [comment]
@ljelen can you rebase onto newest master ?

**Comment by ljelen** [comment]
image
I don't know how to add all .ui files to the package

File: setup.py [hunk 1]
```diff
@@ -52,7 +52,7 @@ def write_version_py(filename='VERSION'):
         # that you indicate whether you support Python 2, Python 3 or both.
         'Programming Language :: Python :: 2.7',
     ],
-    package_data={'pytripgui': ['res/*']},
+    package_data={'pytripgui': ['res/*', './pytripgui/view/*.ui']},
```
Comment (grzanka) [inline setup.py]: Shouldn't it be:
```
package_data={'pytripgui': ['res/*', './view/*.ui']},
```

Read more here https://docs.python.org/3.7/distutils/setupscript.html#installing-package-data

Comment (ljelen) [reply setup.py]: right

File: pytripgui/controller/kernel_cont.py [hunk 2]
```diff
@@ -1,360 +0,0 @@
-import logging
```
Comment (grzanka) [inline pytripgui/controller/kernel_cont.py]: @ljelen why this file is deleted ?

Comment (ljelen) [reply pytripgui/controller/kernel_cont.py]: I forgot to delete it in kernel_vc branch

Comment (grzanka) [reply pytripgui/controller/kernel_cont.py]: OK

**Pull request merged**